cloudtrail.tf

```diff
@@ -15,6 +15,7 @@ resource "aws_cloudtrail" "cloudtrail" {
   include_global_service_events = "${var.include_global_service_events}"
   enable_log_file_validation    = "${var.enable_log_file_validation}"
   is_multi_region_trail         = "${var.is_multi_region_trail}"
+  event_selector                = "${var.event_selector}"
```

Comment:
In the terraform docs it is not that clear (for me) that this attribute is of type list, looks like the type is an object. If the type list works it is fine. Just wondering.

Can you add an example (create a new one based on examples/cloudtrail) which will setup the event selector. Preferable with at least 2 event selectors. We use the examples also to verify the working.